backend/src/users/users.controller.ts

```diff
@@ -67,6 +67,10 @@ export const create = async (req: Request, res: Response, next: NextFunction) =>
   if (!email || !password) {
     return next(new ErrorResponse(errorCode.INVALID_INPUT, status.BAD_REQUEST));
   }
+  const regex = /@student\.42seoul\.kr$/;
+  if (regex.test(email)) {
+    return next(new ErrorResponse(errorCode.INVALID_INPUT, status.BAD_REQUEST));
```

Comment:
```suggestion
    return next(new ErrorResponse(errorCode.STUDENT_42_SUBSCRIPTION_FORBIDDEN, status.BAD_REQUEST));
```

현 상황에서는 새로 추가된 오류 코드인 `STUDENT_42_SUBSCRIPTION_FORBIDDEN`를 추가하는 것이 조금 더 적절할 것 같아요

Reply:
수정하였습니다! 꼼꼼하게 체크해주셔서 감사드립니다!